.gitmodules

```diff
@@ -66,7 +66,7 @@
         url = https://github.com/ESCOMP/CTSM
         fxDONOTUSEurl = https://github.com/ESCOMP/CTSM
         fxrequired = ToplevelRequired
-        fxtag = ctsm5.3.029
+        fxtag = alpha-ctsm5.4.CMIP7.02.ctsm5.3.055
```

Comment:
What a strange tag name...

Reply:
It's a branch tag with the ctsm5.4 dataset.